Lib/tkinter/ttk.py

```diff
@@ -1151,6 +1151,24 @@ def __init__(self, master=None, **kw):
         Widget.__init__(self, master, "ttk::sizegrip", kw)
 
 
+class Spinbox(Entry):
+    """Ttk Spinbox is an Entry with increment and decrement arrows, used
+     for number entry"""
+
+    def __init__(self, master=None, **kw):
+        """Construct a Ttk Spinbox widget with the parent master.
+
+        STANDARD OPTIONS
+
+            class, cursor, style, takefocus
+
+        WIDGET-SPECIFIC OPTIONS
+
+            to, from_, increment
```

Comment:
Ditto adds
values, wrap, format, command
plus "all the features of the ttk::entry widget", which has a few more specific options not listed.